docs/Anjay_integrations/ESP32_integration/M5Stick.md

```diff
@@ -0,0 +1,132 @@
+```
+
+![Client configuration](images/nvs_config.png "Client configuration"){: style="float: left;margin-right: 1177px;margin-top: 17px;"}
+
+## Step 4: Add device to Coiote DM
+
+To connect your M5StickC to the Coiote IoT Device Management LwM2M Server, use your access to a Coiote DM installation, or register at https://www.avsystem.com/try-anjay/ to get access.
+
+To connect the board:
+
+0. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+0. In **Device Inventory**, click **Add device**.
+0. Select the **Connect your LwM2M device directly via the Management server** tile.
+![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+0. In the **Device credentials** step:
+    - In the **Device ID** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+      ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+    - In the **Security mode** section, select the **PSK** mode.
+    - In the **Key identity** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Key** field, type the `psk` key provided in the `nvs_config.csv`.  
+0. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+0. In the **Connect your device** step, wait for the board to connect.
+
+## Step 5: Flash the board and run device
+
+Use pre-built binaries to flash the board and provide credentials by flashing the NVS partition binary.
+
+0. Install the `esptool.py`:
+```
+pip install esptool
+```
+
+0. Flash the board:
+```
+esptool.py -b 750000 --chip esp32 write_flash 0x0000 m5stickc-plus.bin
+```
+
+0. Flash the NVS partition binary:
+```
+esptool.py -b 750000 --chip esp32 write_flash 0x9000 nvs_config.bin
+```
+
+Once executed, the device will be reset and run with the configuration you provided.
+
+   ![Registered device](images/registered_device.png "Registered device")
+
+## M5StickC LwM2M objects
+
+After successful connection to Coiote DM, you can explore the available device objects.
+
+| Target         | Objects
+|----------------|---------------------------------------------
+| ESP32 common   | Security (/0)<br>Server (/1)<br>Device (/3)<br>Firmware Update (/5)<br>WLAN connectivity (/12)
+| M5StickC-Plus  | Push button (/3347)<br>Light control (/3311)<br>Temperature sensor (/3303)<br>Accelerometer (/3313)<br>Gyroscope (/3343)
+
+
+## Upgrade device firmware over the air
+
+To perform a FOTA upgrade, you need an established connection between the M5StickC and Coiote DM (see instructions above).
+
+### Build new firmware version
+
+1. Go to your `anjay-esp32-client` project directory and run `idf.py set-target esp32`.
+2. Run `idf.py menuconfig`, navigate to `Component config/anjay-esp32-client`, and from the supported boards, select **M5StickC**.
+3. Run `idf.py build`.
+4. Once executed, check if the binary file has been built in the following path `$PROJECT_DIR/build/anjay-esp32-client/build/anjay-esp32-client.bin`.
```

Comment:
Maybe I have a different configuration but my binary files are generated in `anjay-esp32-client/build/` directory.